TEditXna/Terraria/World.FileV2.cs

```diff
@@ -558,7 +606,7 @@ public static int SaveTileEntities(World w, BinaryWriter bw)
         {
             bw.Write(w.TileEntities.Count);
 
-            foreach(TileEntity tentity in w.TileEntities)
+            foreach (TileEntity tentity in w.TileEntities)

```

Comment:
There are a lot of auto-formatting changes that make the changes harder to isolate. In the future it makes it easier to keep these as a separate commit or PR. I'm not saying that they shouldn't be done, just that it makes it more difficult to see what actually changed.